pkg/infinispan/client/http/curl/curl.go

```diff
@@ -51,7 +51,7 @@ func (c *CurlClient) Post(podName, path, payload string, headers map[string]stri
 func (c *CurlClient) executeCurlCommand(podName string, path string, headers map[string]string, args ...string) (*http.Response, error, string) {
 	httpURL := fmt.Sprintf("%s://%s:%d/%s", c.config.Protocol, podName, consts.InfinispanPort, path)
 	user := fmt.Sprintf("-u %v:%v", c.credentials.Username, c.credentials.Password)
-	curl := fmt.Sprintf("curl -i --insecure --http1.1 %s %s %s %s", user, headerString(headers), strings.Join(args, " "), httpURL)
+	curl := fmt.Sprintf("curl -i --insecure --digest --http1.1 %s %s %s %s", user, headerString(headers), strings.Join(args, " "), httpURL)
```

Comment:
@ryanemerson is this working for 11.x images? I'm worries about upgrade case

Reply:
Yep. Digest was supported and enabled by default in 11.x. I have manually ran the curl commands against the docker image and it works as expected.

I haven't tried the pre CR1 images, however they should work for the same reason. Regardless CR1 images were Dev releases so we provide no guarantees of compatibility.